src/content/configuration/stats.md

```diff
@@ -607,6 +607,8 @@ module.exports = {
 };
 ```
 
+Set `stats.preset` to `false` tells webpack to use `'none'` [stats preset](/configuration/stats/#stats).
```

Comment:
I am having a hard time parsing this oration, can't understand it at all, sorry

Reply:
Lets try to make it shine, when stats.preset = false, a preset `'none'` is used

Reply:
Any idea on how to improve?

Reply:
```suggestion
Set `stats.preset` to `false` to use `'none'` [stats preset](/configuration/stats/#stats).
```

Reply:
removing `tells webpack` doesnt really add any clarity to the oration, any thoughts @montogeek

Reply:
@EugeneHlushko it certainly does to me. Starting word `Set` doesn't make sense in the proposed change. In suggested change, it makes sense to me. I am leaving another change if `tells webpack` needs to be there.
```suggestion
Setting value of `stats.preset` to `false` tells webpack to use `'none'` [stats preset](/configuration/stats/#stats).
```

Reply:
I was also thinking about that:
```suggestion
Setting value of `stats.preset` to `false` tells webpack to use [stats preset `'none'`](/configuration/stats/#stats).
```

Reply:
Looks good to me 👍,  Just one more suggestion:
The change in order of `'none'` in your suggestion seems counter intuitive to me. For example, we say `Red Car` not `Car red` to describe a car which is `red` in color. Similarly we should call  `'none' stats preset` to describe `stats preset` which is `'none'` in value.

Reply:
Yeah i was thinking it terms of how we set key value pairs, e.g. `stats.preset = 'none'`

Reply:
i'm pretty much happy with `Setting value of `stats.preset` to `false` tells webpack to use `'none'` [stats preset](/configuration/stats/#stats).` lets see if Fernando can contribute to this discussion, elsewise i will add that as is

Reply:
@EugeneHlushko Like you suggest is fine for me

Reply:
```suggestion
Setting value of `stats.preset` to `false` tells webpack to use `'none'` [stats preset](/configuration/stats/#stats).
```